pkgs/tools/graphics/realesrgan-ncnn-vulkan/default.nix

```
@@ -50,15 +50,22 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin $out/share
+
     cp realesrgan-ncnn-vulkan $out/bin/
     cp -r ${models}/models $out/share
   '';
 
+  postFixup = ''
+    patchelf --add-rpath ${lib.makeLibraryPath [ vulkan-loader ]} \
+      --add-needed libvulkan.so \
+      $out/bin/realesrgan-ncnn-vulkan
```

```suggestion
    patchelf $out/bin/realesrgan-ncnn-vulkan --add-needed libvulkan.so
```
I think we just need this

---

Yep, that works! Made the change